app/src/main/java/dev/davwheat/shannonmodemtweaks/tweaks/nvitems/nrcapa/CombosOptimization.kt

```diff
@@ -0,0 +1,24 @@
+package dev.davwheat.shannonmodemtweaks.tweaks.nvitems.nrcapa
+
+import android.os.Parcelable
+import dev.davwheat.shannonmodemtweaks.tweaks.nvitems.NvItem
+import dev.davwheat.shannonmodemtweaks.tweaks.nvitems.NvItemTweak
+import kotlinx.parcelize.IgnoredOnParcel
+import kotlinx.parcelize.Parcelize
+
+@Parcelize
+class CombosOptimization : NvItemTweak(), Parcelable {
+  @IgnoredOnParcel
+  override val name = "NR Combos Optimization"
+
+  @IgnoredOnParcel
+  override val description = "Speeds up UE capability generation and compact them"
```

Comment:
```suggestion
  override val description = "Compacted UE capability response; enabled by default on 8+"
```

think it's important to clarify it's enabled by default on newer generations

Reply:
Okay, but CarrierConfig can disable it, so default is in that sense